addons/udes_stock/wizard/mark_todo_stock_move.py

```diff
@@ -0,0 +1,70 @@
+from odoo import models, fields, api, _
+from odoo.exceptions import UserError
+
+
+class MarkTodoStockMove(models.TransientModel):
+    _name = "mark.todo.stock.move"
```

Comment:
Suggestion: As this wizard deals with pickings directly maybe the name should be something like `stock.picking.mark.todo`, might be worth getting another opinion on this.